react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+
+export default function Stopwatch() {
+  let [time, setTime] = useState(0);
+  let [running, setRunning] = useState(false);
+  const [icon, setIcon] = useState('play');
```

Comment:
You can calculate `icon` from `running`, so it may not need to be kept as state. (Looks like you never set `running` to `undefined`, though, so you'd need to fix that.)